x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_logic.ts

```diff
@@ -70,10 +67,16 @@ export const CrawlerLogic = kea<MakeLogicType<CrawlerValues, CrawlerActions>>({
         onReceiveCrawlerData: (_, { data: { domains } }) => domains,
       },
     ],
-    crawlRequests: [
+    events: [
       [],
       {
-        onReceiveCrawlRequests: (_, { crawlRequests }) => crawlRequests,
+        onReceiveCrawlerData: (_, { data: { events } }) => events,
+      },
+    ],
+    mostRecentCrawlRequest: [
```

Comment:
It seems weird to me that we get the mostRecentCrawl request from the API rather than grabbing it from the list of all events ourselves.  Is there a circumstance in which the value returned by the API for `mostRecentCrawlRequest` would not be the first non-skipped crawl request?

Reply:
The new API in use here returns events, which is a combined list of process crawls and crawl requests. So the list of events could potentially only consist of process crawls, meaning we can no longer compute the most recent crawl request.